src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -57,14 +57,20 @@ function createMutationObserverCallback() {
 function activateHeaderFooter() {
   // Set up elements for the new header and footer
   const headerContainer = document.createElement('div');
+  const skipLink = document.getElementById('skiplink');
   headerContainer.innerHTML = headerPartial;
   headerContainer.classList.add('consolidated');
 
   const footerContainer = document.createElement('div');
   footerContainer.innerHTML = footerPartial;
   footerContainer.classList.add('consolidated');
 
-  document.body.insertBefore(headerContainer, document.body.firstChild);
+  if (skipLink) {
+    document.body.insertBefore(headerContainer, skipLink.nextSibling);
```

Comment:
If we are going to use `skipLink.nextSibling`, we should probably have a check for its existence before referencing it as the node before which we are going to inject.

This could work: 
```
if (skipLink && skipLink.nextSibling) {
   document.body.insertBefore(headerContainer, skipLink.nextSibling);
}
```

But then if `skipLink` is for some reason the only element, `skipLink.nextSibling` will not exist and the header will still be injected _before_ it (via the `else` condition). This situation should never happen, but it's good to be defensive.

The same check should be done for `document.body.firstChild`, arguably.  

Here's what comes to my mind, though I'm open to any fix.
```
  if (skipLink) {
    skipLink.after(headerContainer);
  } else if (document.body.firstChild) {
   document.body.firstChild.before(headerContainer)
  }
```

Reply:
good call, made that adjustment.